### PR TITLE
Migrate license to LGPL-3.0

### PR DESCRIPTION
**Pull request by waltsims**

### CODE REVIEW

File: docs/get_started/license.rst [hunk 1]
```diff
@@ -2,9 +2,29 @@
 License
 =======
 
-GNU GENERAL PUBLIC LICENSE, Version 3, 29 June 2007
-------------
+Current License
```
Comment (faridyagubbayli) [inline docs/get_started/license.rst:5]: Should we add some info about the current license here? Or at least give a link?

**Review by faridyagubbayli — COMMENTED** [review]
Looks great! but I have a comment

File: docs/get_started/license.rst (hunk 1 above)
Comment (waltsims) [reply docs/get_started/license.rst:5]: It is the text box right below. I'll add text for clarification

Comment (faridyagubbayli) [reply docs/get_started/license.rst:5]: Ahh I see now

**Review by faridyagubbayli — APPROVED** [review]
LGTM! Thank you for all this work!

**Pull request merged**